tests.nix

```diff
@@ -36,10 +36,12 @@ let
 
     lint-python = mach-nix.mkPython {
       python = "python39";
+      # Pin these narrowly so that lint rules only change when we specifically
+      # want them to.
       requirements = ''
-        isort
-        black
-        flake8
+        isort == 5.10.1
+        black == 21.12b0
+        flake8 == 4.0.1
```

Comment:
The version of black in the new pypi-deps-db we're updating to has different requirements which are not automatically satisfied by mach-nix.  Rather than try to fix that problem I looked up the versions of these linters we had before and pinned them.